src/common/src/ConfirmDialog/ConfirmDialog.test.tsx

```diff
@@ -148,4 +148,24 @@ describe('<ConfirmDialog />', () => {
 		});
 	});
 	*/
+
+	test('Renders a ReactNode for a confirmation message', () => {
+		jest.isolateModules(async () => {
+			await clickAndRenderDialog(
+				{
+					title: 'Element verwijderen',
+					message: <p data-testid="react-node-message">Weet u zeker dat u dit element wilt verwijderen?</p>,
+					textCancelButton: 'Annuleer',
+					textConfirmButton: 'Verwijder',
+					...callbackMocks,
+				},
+				{},
+			);
+
+			await waitFor(() => {
+				expect(screen.queryByTestId('modal')).toBeInTheDocument();
+				expect(screen.getByText('Weet u zeker dat u dit element wilt verwijderen?')).toBeInTheDocument();
```

Comment:
If it would not render a react node, would it not just render a string, thus would it find the text with getByText?   Perhaps getByTestId('react-node-message') to be in the document?

Reply:
I was actually thinking I should have removed the data-testid attribute after deciding I could just match it on text :slightly_smiling_face: 
The declaration of the react node is in the same code block so I'd argue that is not a concern.